src/app/page.tsx

```diff
@@ -10,6 +13,9 @@ export default function Home() {
         <DepartmentCard type="ped" />
         <DepartmentCard type="rec" />
       </div>
+      <div className="flex justify-center pt-6">
+        <Sociocard />
```

Comment:
```suggestion
        <Sociocard />
```

Since the purpose is to add the colaborator card, why is the component named `SocioCard`.

Which one is with the wrong nomenclature?